### DISCUSSION TITLE
Access bbt 3.5 lcd module sd card

### DISCUSSION THREAD

**Discussion by MichaelGrenier83** [luc-github/ESP3D#1034] [General]
Hi, thanks for all the hard work. I have installed the v3 firmware to access my marlin 3d printer. I use the wifi port on the bigthreetch 3.5 lcd module. I'm unable to access the sd card file that is installed in my lcd bbt module. Is there an option that i need to defind??
Thanks in advance.
Michael

edit: Never mind, so sorry, never saw the option in the dashbord menu to show the tft sd card option.

**Comment by luc-github** [comment]
so it is ok now ?
